src/wagtail_personalisation/rules.py

```diff
@@ -159,7 +159,9 @@ class ReferralRule(AbstractBaseRule):
 
     """
     regex_string = models.TextField(
-        _("Regular expression to match the referrer"))
+        _("Referral"), help_text=_("Enter the domain name (like google.com) "
```

Comment:
The ``.`` characters in domain names may be misinterpreted if the value of this field is going to be interpreted as a regex.